pages.it/common/ln.md

```diff
@@ -5,12 +5,12 @@
 
 - Crea un collegamento simbolico a un file (o directory):
 
-`ln -s {{percorso/al/file}} {{percorso/al/collegamento}}`
+`ln -s {{/percorso/al/file}} {{percorso/al/collegamento}}`
 
 - Sovrascrivi un collegamento esistente in modo che punti a un nuovo file:
 
-`ln -sg {{percorso/al/nuovo/file}} {{percorso/al/collegamento}}`
+`ln -sg {{/percorso/al/nuovo/file}} {{percorso/al/collegamento}}`
```

Comment:
```suggestion
`ln -sf {{/percorso/al/nuovo/file}} {{percorso/al/collegamento}}`
```

I guess we are only adding a `/` so should not be an issue to approve although I do not speak/understand the language.

Also the page for italian does not seem to follow the example from the other pages.

Reply:
Good catch, it seems like a typo. I checked with [deepl](https://www.deepl.com/translator#it/en/Sovrascrivi%20un%20collegamento%20esistente%20in%20modo%20che%20punti%20a%20un%20nuovo%20file) and according to the description, it should be `-sf`.

Reply:
@marchersimon thank you for pointing out deepl. TIL :)